### PR TITLE
Move remaining global post scripts over from UPP

### LINKED ISSUE #630: Update EMC_verif-global tag to verif_global_v2.9.1
**Description**

EMC_verif-global was updated to use hpc-stack modules. See https://github.com/NOAA-EMC/EMC_verif-global/issues/89

The tag version of EMC_verif-global in `sorc/checkout.sh` should be updated to the new `verif_global_v2.9.1` tag.

**Acceptance Criteria (Definition of Done)**

The new tag works in a cycled test on Hera, Orion, and WCOSS.

**Dependencies**

Test on tier 1 supported platforms. Run at least 5 cycles to fully invoke METplus. Verify METplus jobs ran correctly and produced correct output.

@malloryprow for her knowledge

### LINKED ISSUE #769: Move post scripts from UPP under global-workflow
**Description**
Several scripts currently located under NOAA-EMC/UPP are only used for global and should be moved within global-workflow so they can be managed by the global-workflow team.

**Requirements**
All scripts that are global-specific added to this repository. Afterwards, they can presumably be removed from UPP.

**Acceptance Criteria (Definition of Done)**
No scripts copied from UPP.

**Dependencies**
None

### PR DESCRIPTION
**Pull request by WalterKolczynski-NOAA**
**Description**
Several scripts currently located under NOAA-EMC/UPP are only used for global, so they have been moved within global-workflow so they can be managed by the global-workflow team. Scripts are just copied from the current tip of UPP develop, other than some changes to standardize style and indentation.

Fixes #630
Fixes #769

**Type of change**
- [x] New feature (non-breaking change which adds functionality)

**How Has This Been Tested?**
- [x] Forecast-only on Hera
- [x] Coupled on Hera

These changes should also be checked in cycled mode, but still isn't working.
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes

### CODE REVIEW

**Comment by WalterKolczynski-NOAA** [comment]
Going to move forward with these post PRs in global workflow as a sort of transition period. I'll replicate any changes made to these scripts in UPP until the 'official' transition is made.

File: sorc/link_workflow.sh [hunk 1]
```diff
@@ -85,24 +85,33 @@ fi
...
+    # $LINK ../sorc/upp.fd/parm                           post
     [[ -d gldas ]] && rm -rf gldas
     $LINK ../sorc/gldas.fd/parm                              gldas
+cd ${pwd}/../parm/post          ||exit 8
+    # for file in postcntrl_gefs.xml postcntrl_gefs_aerosol.xml postcntrl_gefs_anl.xml postcntrl_gefs_chem.xml \
+    #     postcntrl_gefs_f00.xml postcntrl_gfs.xml postcntrl_gfs_anl.xml postcntrl_gfs_f00.xml postcntrl_gfs_f00_two.xml \
+    #     postcntrl_gfs_flux.xml postcntrl_gfs_flux_f00.xml postcntrl_gfs_goes.xml postcntrl_gfs_goes.xml-new \
+    #     postcntrl_gfs_two.xml postcntrl_gfs_wafs.xml postcntrl_gfs_wafs_anl.xml; do
+    #     $LINK ../sorc/upp.fd/parm/$file .
+    # done
```
Comment (KateFriedman-NOAA) [inline sorc/link_workflow.sh]: Why comment out and not remove?

**Review by KateFriedman-NOAA — APPROVED** [review]
Left one comment. Approve pending successful testing completion.

File: ush/fix_precip.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+   #!/bin/sh
```
Comment (WenMeng-NOAA) [inline ush/fix_precip.sh]: This script is not used in gfs post process.

Comment (WalterKolczynski-NOAA) [reply ush/fix_precip.sh]: Removed it

File: sorc/link_workflow.sh (hunk 1 above)
Comment (WalterKolczynski-NOAA) [reply sorc/link_workflow.sh]: Removed.

**Comment by WenMeng-NOAA** [comment]
@WalterKolczynski-NOAA I assume all gfs UPP control files (under parm/*gfs*xml and parm/*GFS*) are already in the global-workflow, right?

**Comment by WalterKolczynski-NOAA** [comment]
> @WalterKolczynski-NOAA I assume all gfs UPP control files (under parm/_gfs_xml and parm/_GFS_) are already in the global-workflow, right?

They weren't on the list of files to bring over in the presentation you gave us, and since they are used for the model, I thought you might want to keep them under UPP. Only the four global_* files that are part of this PR have been copied over so far; we're linking the rest. The xml files don't appear to be used at all during normal GFS operation (just to generate new flat files, maybe?), so I wasn't even linking those.

We can bring over whichever files you think appropriate: either now. or later when you are ready to handle the other end of this.

**Comment by WenMeng-NOAA** [comment]
The UPP flat text control  files will be used in the runtime. The control xml files are human readable format for variables output from the UPP. Adding new products will be started from control files. It would be your option for keeping control files at both repositories or linking from the UPP repository.

**Comment by WalterKolczynski-NOAA** [comment]
Thanks, that helps. We'll keep linking them. One thing we definitely don't want to do is have two different 'canonical' copies, so as long as there is need to keep those files in upp, we won't also have a copy in our own repo. For any that may only be needed by global workflow, we can revisit later.

**Pull request merged**